src/flow.c

```diff
@@ -580,8 +580,7 @@ void FlowInitConfig(bool quiet)
             FatalError(SC_ERR_FATAL, "Invalid value for flow.hash-size: NULL");
         }
 
-        if (StringParseUint32(&configval, 10, strlen(conf_val),
-                                    conf_val) > 0) {
+        if (StringParseUint32(&configval, 10, (uint16_t)strlen(conf_val), conf_val) > 0) {
```

Comment:
while I'm generally not a fan of `size_t`, I wonder if we should use it here as we'd most of the time pass the result of `strlen()`?

Reply:
Done in https://github.com/OISF/suricata/pull/7239